app/Http/Transformers/PostTransformer.php

```diff
@@ -27,6 +27,7 @@ public function transform(Post $post)
                 ],
                 'caption' => $post->content->caption,
                 'status' => $post->content->status,
+                'total_reactions' => count($post->content->reactions),
```

Comment:
This may secretly still be making an extra SQL query – if a relationship isn't [eager loaded](https://laravel.com/docs/5.4/eloquent-relationships#eager-loading) it'll only make the call for the "related" model once it's requested, which would still mean we might only load reactions once we render the `PostTransformer`.

It could be helpful to add some logging to the [database query event](https://laravel.com/docs/5.4/database#listening-for-query-events) so we can see exactly what database queries are made on a typical request.